### PR TITLE
add-saving-apartment-entry-date-and-neighborhood-to-firebase

### PR DESCRIPTION
**Pull request by amit-schwartz-u**
add-saving-apartment-entry-date-and-neighborhood-to-firebase

### CODE REVIEW

**Review by ShaharBirzon — APPROVED** [review]
אחלה!

File: app/src/main/java/com/example/roome/Roommate_searcher_tabs_classes/EditProfileRoommateSearcher.java [hunk 1]
```diff
@@ -137,6 +138,13 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         return inflater.inflate(R.layout.activity_edit_profile_roommate_searcher, container, false);
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        setFieldsToValuesStoredInFirebase();
+        validateUserInput();
```
Comment (levYuval1007) [inline app/src/main/java/com/example/roome/Roommate_searcher_tabs_classes/EditProfileRoommateSearcher.java:145]: why moved from onActivityCreatd?

File: app/src/main/java/com/example/roome/Roommate_searcher_tabs_classes/EditProfileRoommateSearcher.java [hunk 2]
```diff
@@ -149,6 +157,7 @@ void initializeFirebaseFields() {
     private void initializeFields() {
         homeNeighborhood = getView().findViewById(R.id.spinner_neighborhood);
         displayDate = getView().findViewById(R.id.iv_choose_apartment_entry_date);
+        chosenDate = getView().findViewById(R.id.tv_show_here_entry_date);
```
Comment (levYuval1007) [inline app/src/main/java/com/example/roome/Roommate_searcher_tabs_classes/EditProfileRoommateSearcher.java:160]: great!

**Review by levYuval1007 — CHANGES_REQUESTED** [review]
good work!

**Pull request merged**